openapi/src/main.rs

```diff
@@ -1,30 +1,51 @@
 use std::{collections::BTreeSet, fs};
 
 use anyhow::{Context, Result};
+use structopt::StructOpt;
 
+use crate::spec_fetch::fetch_spec;
 use crate::{metadata::Metadata, url_finder::UrlFinder};
 
 mod codegen;
 mod file_generator;
 mod mappings;
 mod metadata;
+mod spec_fetch;
 mod types;
 mod url_finder;
 mod util;
 
+#[derive(Debug, StructOpt)]
```

Comment:
Used `structopt` instead of `clap` because `clap` (and its deps require the 2021 edition)

Reply:
:+1: